src/PKSim.Core/Services/PopulationPKAnalysis.cs

```diff
@@ -7,11 +7,13 @@ public class PopulationPKAnalysis
    {
       public CurveData<TimeProfileXValue, TimeProfileYValue> CurveData;
       public PKAnalysis PKAnalysis { get; set; }
+      public string ExtraDescription { get; private set; }
 
-      public PopulationPKAnalysis(CurveData<TimeProfileXValue, TimeProfileYValue> curveData, PKAnalysis pkAnalysis)
+      public PopulationPKAnalysis(CurveData<TimeProfileXValue, TimeProfileYValue> curveData, PKAnalysis pkAnalysis, string extraDescription = "")
```

Comment:
string,empty

Reply:
Compiler complains, it needs to be a compile time constant